user-console/source/org/pentaho/mantle/client/ui/tabs/MantleTabPanel.java

```diff
@@ -88,6 +96,7 @@ public void onWindowClosing( ClosingEvent event ) {
       }
     } );
 
+    setIsAsyncModeEnabled();
```

Comment:
I see. But please rename the function so that it does not try to look like a setter.